pretty_cron/api.py

```diff
@@ -1,10 +1,15 @@
+# -*- coding: utf-8 -*-
 import datetime
 
 
 def prettify_cron(expression):
+    """
+    Returns human readable cron-tab configuration
+    """
     pieces = []
-    for i, piece in enumerate(expression.split(" ")):
-        if i == 3 and ',' in piece:  # support comma-separated values for month
+    for piece in expression.split(" "):
+        # support comma-separated values for ordinal days, weekdays and months
```

Comment:
if this only supports days, weekdays, and months, shouldn't the conditional include, like before, `if i in (1, 2, 3)` ? otherwise you're allowing `,` in hours